meetings/2023-01-04.md

```diff
@@ -29,36 +39,63 @@
 
 
 ### Scribes
-* name
+* April Daly
 
 
 ### Introductions
-* name: text
+* Skipped - all participants have attended this group meeting in the past.
 
 ---
 
 
 ## Topics
 
+
+### Index and Registry
+* SC: Registry as per W3C Process: https://www.w3.org/2021/Process-20211102/#registry
+* SC: In order to document collections of values or other data that TRs use, a separate document can be published. For example, https://solidproject.org/TR/notifications-protocol refers to https://solidproject.org/TR/notification-subscription-types (Living Document) where an index of subscription types can be found.
+* SC: To date, the index document only listed documents published under `TR/`.
+* SC: The notifications-panel also has https://github.com/solid/notifications/blob/main/README.md#subscription-types that lists all types.
+* SC: Question: Would it be simpler to re-use https://solidproject.org/TR/ as the index for auxiliary documents (e.g., notification channel types), since it is intended to list all of the CG's Work Items? The "Indexing Procedure" can be part of TRs that need it.
+* SC: Quick Overview: a document that would hold things that need to be registered and are typically dependent on other specifications. Notifications protocol allows auxiliary specs that define individual notification channel types, for example.  Similar to [DID method approach](https://www.w3.org/TR/did-spec-registries/#did-methods).
+* We have published web socket subscription type (with corresponding document under TR). Terminology has been changed, from *Subscription types* to *channel types*. It would be incorrect to rely on notification subscription types. If this approach is continued a new document set is needed.
+* SC: Instead of recreating documents or reusing, we could use the solid/TR, regardless of maturity. The TR document would be the central document with links to all related documents. 
+* PC: Suggest W3C doesn't require a separate document, but this should be verified. Propose using the TR document as a registry of SOLID and related specifications. Registries could be considered different (i.e., entries of a given registry are homogeneous). A registry is defined by a process. The TR Index document list has a heterogeneous mix of documents. We could use that document to hold all the registries that are need, but it would be ideal if it structured.
```

Comment:
```suggestion
* PC: I believe W3C doesn't require a separate document, but this should be verified. Propose using the TR document as a registry of Solid and related specifications. Registries could be considered different (i.e., entries of a given registry are homogeneous). A registry is defined by a process. The TR Index lists a heterogeneous mix of documents. We could use that document to hold all the registries that are needed, but it would be ideal if it structured.
```